### PR TITLE
Feature/31 update password

### PR DESCRIPTION
**Pull request by jillianlg**

### CODE REVIEW

File: sql/setup.sql [hunk 1]
```diff
@@ -19,7 +19,7 @@ CREATE TABLE tips (
 );
 
 INSERT INTO users (email, password_hash) VALUES
-('admin@behuman.com', '$2y$04$F8I5POR7CFJR1HgpKeuvs.jYH6Ge60paGorKoQxvlhzWjoZoZGkBi');
+('behumannotbot@gmail.com', '$2y$04$F8I5POR7CFJR1HgpKeuvs.jYH6Ge60paGorKoQxvlhzWjoZoZGkBi');
```
Comment (bryanakitchen) [inline sql/setup.sql:22]: Should this be protected and placed in .dotenv file?

**Review by bryanakitchen — APPROVED** [review]
Left one comment.

File: lib/utils/email.js [hunk 2]
```diff
@@ -9,22 +9,22 @@ const transporter = nodemailer.createTransport({
...
-  subject: 'test email', // Subject line
-  text: 'this is a test of the admin invite link.', // plain text body
-  html: '<a href="http://api/v1/auth/invite">Invite Link</a>', // html body
+  const invite = {
+    from: '"NotBot Admin" <behumannotbot@gmail.com>', // sender address
+    to: email,  // list of receivers
+    cc: 'behumannotbot@gmail.com', 
+    subject: 'beHuman Admin Invite', // Subject line
+    text: 'admin invite link.', // plain text body
+    html: 'Use this <a href="https://be-human-demo-staging.herokuapp.com">Invite Link</a> to access the admin page. You will need to use the password: adminPassword and this email to login.', // html body
```
Comment (ShaneUP1) [inline lib/utils/email.js]: So can anyone go to this link (our heroku link) and type in adminPassword and login? Or will this only work for our specific email addresses?

**Review by ShaneUP1 — APPROVED** [review]
Left one comment but looks good!

File: lib/utils/email.js (hunk 2 above)
Comment (jillianlg) [reply lib/utils/email.js]: Won't work for just anyone - I can explain why during our stand up today. :)

File: sql/setup.sql (hunk 1 above)
Comment (jillianlg) [reply sql/setup.sql:22]: Good question - I had that thought too. Shouldn't be an issue as you can not log in with the token. You log in on the front end with the password that the token represents. However, still worth asking about!

**Pull request merged**